README.md

```diff
@@ -82,7 +83,12 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+::= "the"
+::= "smelly" | "lazy"
+::= "dog" | "cat"
+::= "ate" | "ran"
+::= "slowly" | "noisily" | ""
+::= | ""
```

Comment:
This isn't BNF, as there are no nonterminals on the left of the ::=. It also doesn't capture the repeating nature of the adjective. 

I was looking for something like

~~~
  <sentence> ::= "the" <adjectives> <noun> <verb> <adverb>

  <adjectives> ::= "" | <adjectives> <adjective>

  <adjective> ::= "lazy" | "smelly"

  <noun> ::= "dog" | "cat"

  <verb> ::= "ate" | "ran"

  <adverb> ::= "slowly" | "noisily" | ""
~~~